sqlite-conf.sql

```diff
@@ -0,0 +1,21 @@
+DROP TABLE ftpctxt;
+CREATE TABLE ftpctxt (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    parent_id INTEGER UNSIGNED,
+    name VARCHAR(255),
+    type VARCHAR(255),
+    info VARCHAR(255)
+);
+
+DROP TABLE ftpconf;
+CREATE TABLE ftpconf (
```

Comment:
Why not use `CREATE TABLE ... IF NOT EXISTS`, instead of using `DROP TABLE ...` beforehand?